### PR TITLE
Update Dockerfile and reset the administrative password instructions

### PR DESCRIPTION
**Pull request by Jonnyan404**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,6 +26,6 @@ WORKDIR /usr/local/memos
 COPY --from=backend /backend-build/memos /usr/local/memos/
 
 # Directory to store the data, which can be referenced as the mounting point.
-RUN mkdir -p /var/opt/memos
+RUN mkdir -p /var/opt/memos && apk add --no-cache sqlite
```
[the remark this answers is not in the document]
Comment (Jonnyan404) [reply Dockerfile:29]: > Please move the new command into a new line.
> 
> e.g. https://github.com/usememos/memos/blob/main/Dockerfile#L15

This way is to reduce the Docker mirror layer, thereby narrowing the final mirror size.

[the remark this answers is not in the document]
Comment (boojack) [reply Dockerfile:29]: Please check the updated comment.

[the remark this answers is not in the document]
Comment (Jonnyan404) [reply Dockerfile:29]: > I think it's unnecessary to install sqlite in docker image. Maybe it's better to put the reset password commands into GitHub discussions. Then add a link in the README.

That's okay

[the remark this answers is not in the document]
Comment (Jonnyan404) [reply Dockerfile:29]: > I think it's unnecessary to install sqlite in docker image. Maybe it's better to put the reset password commands into GitHub discussions. Then add a link in the README.

https://github.com/usememos/memos/discussions/939

**Pull request closed without merging**